### DISCUSSION TITLE
[FEATURE REQUEST] Allow iron doors, and trapdoors to be opened by hand.

### DISCUSSION THREAD

**Discussion by KayaEdwin** [PurpurMC/Purpur#990] [__Dismissed__] [answered]
As there's an option to make it so normal doors can only be opened by redstone, I think adding this would be a perfect addition.

**Comment by katekerllenevich** [comment, marked as answer]
Unfortunately this option seems to be restricted client side, foiled again!

**Reply by YouHaveTrouble** [reply]
Are you sure about that? https://github.com/YouHaveTrouble/PurpurExtras/commit/cfae81fc25b5397d78e9e56b846e70ff7cc49baf

**Reply by katekerllenevich** [reply]
When removing the restriction on the server and right clicking on the door, nothing happened. I might experiment with this more later.
